### PR TITLE
origin/computed_properties_mobility

### PR DESCRIPTION
**Pull request by ArtemFediai**
mobility property added, merged to main_init

### CODE REVIEW

File: docs/source/user_guide/computed_properties.rst [hunk 1]
```diff
@@ -510,23 +510,132 @@ Data Files in `Analysis/DOS/data`
...
-.. Charge Carrier Mobility
-.. -----------------------
+Charge Carrier Mobility
+-----------------------
+
+Charge carrier mobility is a measure of the speed at which charge carriers, such as electrons or holes,
+can move through a material when an electric field is applied. It reflects the material's ability to conduct charge and
+is influenced by various intrinsic and extrinsic factors.
+
+
```
Comment (tneumann2021) [inline docs/source/user_guide/computed_properties.rst:520]: Should we stick to the pattern of the rest of the properties and do 

Property description
^^^^^^^^^^^^^^^^^^^^^
[everything up to Output files] 

Output files
^^^^^^^^^^^^^^^

I think the levels of headers are === --- ^^^ ~~~~

File: docs/source/user_guide/computed_properties.rst [hunk 2]
```diff
@@ -510,23 +510,132 @@ Data Files in `Analysis/DOS/data`
...
+where constants are:
+
+- :math:`e` is the elementary charge,
+- :math:`\beta = \frac{1}{k_B \cdot T}` with :math:`k_B` being the Boltzmann constant and :math:`T` being the inverse temperature,
+- :math:`n` is the dimensionality of the system,
+- :math:`\hbar` is the reduced Planck's constant,
+
+Other parameters are explained below.
+
+Fixed Parameters
```
Comment (tneumann2021) [inline docs/source/user_guide/computed_properties.rst:541]: Subsection of "Modeling Method"? If so, header should be one level lower, same for Computed Parameters

File: docs/source/user_guide/computed_properties.rst [hunk 3]
```diff
@@ -510,23 +510,132 @@ Data Files in `Analysis/DOS/data`
...
+
+Computed Parameters
+~~~~~~~~~~~~~~~~~~~
+
+Parameters below are highly materials-specific and are computed with `Nanoscope`:
+
+- :math:`\sigma`: Energy disorder due to environmental variations in molecular energy levels, [:math:`eV`],
+- :math:`J^2 r^2`: Reflects the ease of charge transfer between molecules,  [:math:`eV²·Å²`].
+
+**Computed Mobility Data**
```
Comment (tneumann2021) [inline docs/source/user_guide/computed_properties.rst:559]: Should that not be part of Output Files?

File: docs/source/user_guide/computed_properties.rst [hunk 4]
```diff
@@ -510,23 +510,132 @@ Data Files in `Analysis/DOS/data`
...
-.. Content about Mobility.
 
 
 .. _overlap_integrals:
 
 .. Overlap Integrals
 .. -----------------
 
-.. Content about Overlap Integrals.
+Content about Overlap Integrals.
```
Comment (tneumann2021) [inline docs/source/user_guide/computed_properties.rst:633]: These I commented for the purpose that we do not have empty headers, until we find time to fill

File: docs/source/user_guide/computed_properties.rst [hunk 5]
```diff
@@ -510,23 +510,132 @@ Data Files in `Analysis/DOS/data`
...
 
 
 .. _energy_disorder:
 
-.. Energy Disorder
-.. ---------------
+Energy Disorder
```
Comment (tneumann2021) [inline docs/source/user_guide/computed_properties.rst:638]: Same as above

File: docs/source/user_guide/computed_properties.rst [hunk 6]
```diff
@@ -510,23 +510,132 @@ Data Files in `Analysis/DOS/data`
...
+---------------
 
-.. Content about Energy Disorder.
+Content about Energy Disorder.
```
Comment (tneumann2021) [inline docs/source/user_guide/computed_properties.rst:641]: Same as above

**Review by tneumann2021 — CHANGES_REQUESTED** [review]

File: docs/source/user_guide/computed_properties.rst (hunk 1 above)
Comment (ArtemFediai) [reply docs/source/user_guide/computed_properties.rst:520]: fixed

File: docs/source/user_guide/computed_properties.rst (hunk 2 above)
Comment (ArtemFediai) [reply docs/source/user_guide/computed_properties.rst:541]: yes. fixed

File: docs/source/user_guide/computed_properties.rst [hunk 7]
```diff
@@ -208,22 +208,60 @@ In addition, this module provides the following output files:
...
+
+Modeling Method
+~~~~~~~~~~~~~~~
+`Nanoscope` calculates the dielectric permittivity using the polarization energy as a function of the polarization shell radius.
+For details, refer to the paper:
+
+  1. :ref:`science_publications_ipea_eps_paper`,
+
+
+Output Files
```
Comment (tneumann2021) [inline docs/source/user_guide/computed_properties.rst:236]: Wrong level of header for output files: ^^^^^^^^^^^

**Pull request merged**